### DISCUSSION TITLE
How to change window size in main.rs?

### DISCUSSION THREAD

**Discussion by timothy-mayweather** [tauri-apps/tauri#3225] [Q&A] [answered]
I have used the window functions e.g window.set_title() alot but have failed completely on the window.set_size function. Could i get help please? Thanks

**Comment by amrbashir** [comment, marked as answer]
Provide some code, what have you tried? what do you expect? any errors?

**Reply by FabianLars** [reply]
Just so i can mark this as resolved, i'm gonna give a one-liner example:
```rs
window.set_size(Size::Logical(LogicalSize { width: 100.0, height: 100.0 })).unwrap();
// or
window.set_size(Size::Physical(PhysicalSize { width: 100, height: 100 })).unwrap();
// btw don't use .unwrap() here, i'm just lazy.
```

**Reply by timothy-mayweather** [reply]
Sorry, I solved it and didnt come back to update. Thanks

**Reply by wangtiger317** [reply]
> Just so i can mark this as resolved, i'm gonna give a one-liner example:
> 
> ```rust
> window.set_size(Size::Logical(LogicalSize { width: 100.0, height: 100.0 })).unwrap();
> // or
> window.set_size(Size::Physical(PhysicalSize { width: 100, height: 100 })).unwrap();
> // btw don't use .unwrap() here, i'm just lazy.
> ```

not working at all
